lib/geckoboard/dataset.rb

```diff
@@ -16,6 +16,10 @@ def put(data)
       client.put_data(id, payload_formatter.format(data))
     end
 
+    def post(data, options={})
```

Comment:
🏄  I'm wondering if "post" is the right name for this concept? would "append" feel more natural?
No doubt there's already been some discussion around this 😄